keyboards/walletburner/cajal/rules.mk

```diff
@@ -18,3 +18,4 @@ BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
 ENCODER_ENABLE = yes    # Enable support for EC11 Rotary Encoder
+VIA_ENABLE = yes
```

Comment:
please remove.  This should only be defined at the keymap level. 
```suggestion
```

Reply:
Understood